dvc/config_schema.py

```diff
@@ -145,6 +145,7 @@ class RelPath(str):
                     "session_token": str,
                     Optional("listobjects", default=False): Bool,  # obsoleted
                     Optional("use_ssl", default=True): Bool,
+                    Optional("verify_ssl", default=True): Bool,
```

Comment:
While we are at it, we should also try to provide a way to specify a custom cert (see https://github.com/iterative/dvc/issues/5388). We could just use `verify_ssl` to accept a string to a path or create a separate config. 

Note that we have the `ssl_verify` option in HTTP remote already, so we are being inconsistent here. We should deprecate either of those. 🙂

Reply:
I agree with your first point, but I disagree with the second. As we are directly parsing this to boto3, here is what they say about the difference: 

- use_ssl (boolean) -- Whether or not to use SSL. By default, SSL is used. Note that not all services support non-ssl connections.

- verify (boolean/string) -- Whether or not to verify SSL certificates. By default SSL certificates are verified. You can provide the following values:
**False** - do not validate SSL certificates. SSL will still be used (unless use_ssl is False), but SSL certificates will not be verified.
**path/to/cert/bundle.pem** - A filename of the CA cert bundle to uses. You can specify this argument if you want to use a different CA cert bundle than the one used by botocore.

Reply:
@NigelVanHattum, not sure I understand. I am saying that maybe we should extend `verify_ssl` to also accept the path to the certificates (as you have written, they accept boolean and the path to cert, requests work in a similar fashion).

Reply:
@skshetry Good point about `ssl_verify`, totally forgot about it. @NigelVanHattum would you be so kind to rename it once more, please? Sorry for the inconvenicence. 

Cert path option could probably wait until someone asks for it.

Reply:
> Cert path option could probably wait until someone asks for it.

@efiop, users are asking about this. We have been pointing users to `AWS_REQUESTS_BUNDLE` and `REQUESTS_CA_BUNDLE` for quite a long time now. They do work, but it always feels like a hack.

Reply:
I am not trying to block this PR, it could be introduced separately. This config does have its place, I just worry that we'll suggest this to the users instead of a more secure option in which the certificates are validated properly.

Reply:
Just added the option to give a path as a value to this parameter as well. I will change the name to ssl_verify

Reply:
> @NigelVanHattum, not sure I understand. I am saying that maybe we should extend `verify_ssl` to also accept the path to the certificates (as you have written, they accept boolean and the path to cert, requests work in a similar fashion). 🙂

Aah, I think I've read your comment wrong. I'm renaming the parameter